lib/elixir/lib/keyword.ex

```diff
@@ -548,9 +552,9 @@ defmodule Keyword do
   defp delete_key([], _key), do: []
 
   @doc """
-  Deletes the first entry in the keyword list for a specific `key`.
+  Deletes the first entry in the keyword list under a specific `key`.
 
-  If the `key` does not exist, returns the keyword list unchanged.
+  If the `key` does not exist, the keyword list is returned unchanged.
```

Comment:
We should avoid passive voice:

```suggestion
  If the `key` does not exist, it returns the keyword list unchanged.
```